doc/kubernetes/modules/ROOT/pages/openshift/cross-site-rosa.adoc

```diff
@@ -70,7 +70,22 @@ For Keycloak deployment, check xref:customizing-deployment.adoc[Keycloak Customi
 Note that not all variable are respected.
 As an example, `KC_ISPN_NAMESPACE` is not possible to change since it is automatically computed by this installation script.
 
-=== Warnings / Known issues
+=== Customize Keycloak source
 
-Using `--force` will break the keycloak deployment because it patches the keycloak operator deployment with an incorrect image.
+This setup, by default, deploys Keycloak from nightly build.
+To create a deployment for specific Keycloak source code, git repository and branch can be specified.
+This replaces deployments for both Keycloak and Keycloak operator.
+Specify the following variables for using custom source code:
+
+
+|===
+|Variable |Details
+
+|KC_REPOSITORY
+|Git repository to clone Keycloak source code from.
```

Comment:
it would be nice to provide an example value for reference, it helps clear the confusion which URI should the user provide.